go/client/cmd_ctl_stop.go

```diff
@@ -36,6 +36,19 @@ func (s *CmdCtlStop) ParseArgv(ctx *cli.Context) error {
 }
 
 func (s *CmdCtlStop) Run() (err error) {
+	configCli, err := GetConfigClient(s.G())
+	if err != nil {
+		return err
+	}
+	config, err := configCli.GetConfig(context.TODO(), 0)
+	if err != nil {
+		return err
+	}
+
+	if config.ForkType == keybase1.ForkType_LAUNCHD {
+		return StopLaunchdService(s.G(), config.Label, true)
```

Comment:
Its safer to call stop from the client instead of asking the service to call stop..

The difference is, the old code was calling launchd stop from the service. The new code the launchd stop is called from the client only. (As a reminder launchd stop execs `launchctl remove <label>` which does a term signal on the service.

This also has a nice benefit of removing some service code. I think it's a good change?